app/globals.css

```
@@ -1,59 +1,74 @@
-}
-
-@layer base {
-  * {
-    @apply border-border;
-  }
-  body {
-    @apply bg-background text-foreground;
-  }
+@import 'tailwindcss';
+
+@theme inline {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)